stopify/test-data/integration/stack-trace-external.html

```diff
@@ -0,0 +1,52 @@
+<html>
+
+<body>
+  <p>This page runs Stopify in the browser.</p>
+<script src="../../dist/stopify-full.bundle.js"></script>
+<script>
+
+function Ext() {
+  Ext2();
+}
+
+function Ext2() {
+  throw 'bad';
+}
+
+var program = `function F() {
+    Ext();
+}
+
+function G() {
+  F();
+}
+
+G();
+`;
+
+var runner = stopify.stopifyLocally(program, { externals: [ 'Ext' ] });
+runner.run((err) => {
+  console.log(err.stack.join("\n"));
+
+  // NOTE(arjun): This test case may appear over-specified. But, I believe that
+  // for a trivial example like this, we should be able to write the stack
+  // trace we want.
+  if (err.value.toString().indexOf('bad') >= 0 &&
```

Comment:
@jpolitz you may find this example interesting. Here, an exception is thrown deep in external code, but the reported stack trace only has locations in the stopified code (including the position where it calls the external code).